core/src/main/java/org/sonarsource/sonarlint/core/fs/FileExclusionService.java

```diff
@@ -114,6 +115,12 @@ public void onFileSystemUpdated(FileSystemUpdatedEvent event) {
     event.getAddedOrUpdated().forEach(f -> serverExclusionByUri.invalidate(f.getUri()));
   }
 
+  @EventListener
+  public void onFileExclusionSettingsChanged(FileExclusionChangedEvent event) {
```

Comment:
```suggestion
  public void onSonarServerSettingsChanged(SonarServerChangedEvent event) {
```
I think the event should represent a change of the settings on the Sonar server side.